### DISCUSSION TITLE
Is there a way to install Amule or Andydesk?

### DISCUSSION THREAD

**Discussion by DEMIANDOR** [itsManjeet/avyos#28] [Q&A] [answered]
Is there a way to install Amule or Andydesk?
I really like this operating system but, it would be much more relaxed if I could install software like on Manjaro or Garuda. Thanks so much!!

**Comment by itsManjeet** [comment, marked as answer]
Hey, Anydesk is remove for flathub https://github.com/flathub/com.anydesk.Anydesk/issues/109, since flatpak is the default package management.

Alternatively, You can use distrobox to install anydesk

download the .deb from Anydesk site
`$ distrobox enter ubuntu (or your container-name)`
and not inside distrobox
`$ sudo apt install libpolkit-gobject-1-0`
`$ dpkg -i Anydesk_XX.deb`
`$ sudo apt install -f`

Now to integerate it into the rlxos
`$ distrobox-export --app anydesk`

**Reply by DEMIANDOR** [reply]
Thank you so much. I already have Anydesk working now.

**Comment by Ndashka-Dev** [comment]
While **I do not recommend** EOL software, especially for corporate environments, there is an option to install Anydesk from Flathub. You simply need to open the terminal and type in "_flatpak install com.anydesk.Anydesk_", it will inform you, that it is no longer maintained, but will then proceed to install it as normal. Another option is to use an Appimage. There are plenty of them out there, for example, [this](https://github.com/ivan-hc/Anydesk-appimage).

Alternatively, you could use an another remote control software, such as [Rustdesk](https://rustdesk.com). In my opinion, it's one of the best opensource alternatives out there. It also has an Appimage release.

**Reply by DEMIANDOR** [reply]
Thank you so much. I already have Anydesk working now. And I take into account all your suggestions.
